kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -86,6 +86,7 @@ kieserver.dmn.enabled=true
 kieserver.jbpm.enabled=true
 kieserver.jbpmui.enabled=true
 kieserver.casemgmt.enabled=true
+#kieserver.prometheus.enabled=true
```

Comment:
Here we're adding the commented line as a convenience to the user, right? if they need it, they just remove the hash sign ...

Reply:
correct